test/built-ins/Temporal/Calendar/prototype/daysInMonth/throw-type-error-RequireInternalSlot.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
```

Comment:
Consider copying the `branding.js` test from https://github.com/tc39/proposal-temporal/blob/main/polyfill/test/Calendar/prototype/daysInMonth/branding.js which covers this case more comprehensively. (Same remark for the other PRs)

Reply:
I've created an issue for @FrankYFTang to follow up on.

Reply:
I would love to see temporal champion to move their test to test262 sooner than later so I could focusing on developing code on v8 instead of adding more tests. If Temporal have better tests, it will be nice if they can contribute and move them into test262 so all the browser engine integrated with test262 could take advantage of them. I prefer whoever owning those tests in temporal take charge of that instead of asking me who try to fill the gap now to do that.

Reply:
I outlined the plan for moving them over in https://github.com/tc39/test262/issues/3002#issuecomment-887908934
Ms2ger is working on it, but is not available this week.

If I tried to cut corners and move everything over right now, the tests would start to diverge and it would be really inconvenient for ensuring that we correctly and completely converted the legacy test suite. If we accidentally dropped test coverage during this process, that would be a worse outcome in the end for engines. I realize this is inconvenient in the short term, but for this reason I'd really prefer to stick to the plan that I already outlined.

There shouldn't be a problem with moving over the `branding.js` tests since I don't expect anyone will really need to revise or change those. If you don't want to do it, that's fine, one of the maintainers can assign #3137 to me.

I also think it should be fine, when you are contributing tests, to just take a look there to check if it is already done — no need to spend any effort on duplicating tests that we will move over here ASAP anyway.